docs/manual/format.md

```diff
@@ -48,31 +51,31 @@ struct rpmlead {
 };
 \endcode
 
-and is illustrated with one pulled from the rpm-2.1.2-1.i386.rpm
-package:
+and is illustrated with one pulled from the rpm-2.1.2-1.i386.rpm package:
 
 ```
 00000000: ed ab ee db 03 00 00 00
 ```
 
-The first 4 bytes (0-3) are "magic" used to uniquely identify an RPM
-package.  It is used by RPM and file(1).  The next two bytes (4, 5)
-are int8 quantities denoting the "major" and "minor" RPM file format
-version.  This package is in 3.0 format.  The following 2 bytes (6-7)
-form an int16 which indicates the package type.  As of this writing
-there are only two types: 0 == binary, 1 == source.
+The first 4 bytes (0-3) are the "magic" number used to uniquely identify a file
+as an RPM package.  It is used by RPM and file(1).  The next two bytes (4, 5)
+are int8 quantities denoting the "major" and "minor" RPM file format version.
+For legacy reasons, this version is always "3.0" (major version "3", minor
+version "0"), even with packages built by RPM 4.0+ (referred to as RPM v4
+packages). The following 2 bytes (6-7) form an int16 which indicates the
+package type.  As of this writing there are only two types: 0 == binary,
+1 == source.
 
 ```
 00000008: 00 01 72 70 6d 2d 32 2e    ..rpm-2.
 ```
 
-The next two bytes (8-9) form an int16 that indicates the architecture
-the package was built for.  While this is used by file(1), the true
-architecture is stored as a string in the Header.  See, lib/misc.c for
-a list of architecture->int16 translations.  In this case, 1 == i386.
```

Comment:
I couldn't find that mapping anymore, but I don't know if we have much need to discuss it anyway.

Reply:
No wonder, the arch translations have been in the declarative rpmrc config file for over twenty years. This is one of the docs thats *so old* it makes you wonder if its worth trying to patch it up...